.github/workflows/build-documentation.yml

```diff
@@ -60,6 +60,11 @@ jobs:
         sudo apt-get update
         sudo apt-get install mono-complete --yes
 
+    # Note:
+    # Since we use a custom template to override some properties of the docfx default template, when upgrading docfx we should check
```

Comment:
Is it possible to just add out own custom CSS that overrides what's needed, instead of duplicating the docfx one?

Reply:
Maybe.
At least that's not the standard docfx way (see https://dotnet.github.io/docfx/tutorial/walkthrough/advanced_walkthrough.html).
The templates are merged on a file by file base and as I understand it, there's no way to "patch" a file when merging.
If you choose to change a single line in a template file you always have copy over (and maintain) the whole file.

Reply:
Oh, I didn't mean to patch anything - just to reference an additional (static) CSS file from the HTML template, which would contain only the stuff we need to override/tweak (like how HTML/CSS usually works)

Reply:
Yeah, probably.
The problem would still apply in that case. It'd be just another template file we merge.

Reply:
OK. It may be worth not copying the entire docfx CSS, since it might evolve and it's better to not think about it... And having our minimal overriding CSS also makes it very clear what exactly we're tweaking etc.

But if it's a problem we can also copy out the CSS, I don't any strong objections.

Reply:
I'll look into it.

Reply:
Ok, that works and is indeed a better solution.